test/gie/4D-API_cs2cs-style.gie

```diff
@@ -0,0 +1,132 @@
+but provided through a different interface.
+
+===============================================================================
+
+
+<gie>
+
+-------------------------------------------------------------------------------
+Test the handling of the +towgs84 parameter
+-------------------------------------------------------------------------------
+This example is from Lotti Jivall: "Simplified transformations from
+ITRF2008/IGS08 to ETRS89 for maritime applications". The original
+XYZ data (cf. more_builtins.gie) have been transformed to LPZ using
+this command: echo x_val y_val z_val | cct -It0 proj=cart ellps=GRS80 --
+-------------------------------------------------------------------------------
+operation proj=latlong ellps=GRS80
+          towgs84 =  0.676780,  0.654950, -0.528270,
+                    -0.022742,  0.012667,  0.022704, -0.010700
+-------------------------------------------------------------------------------
+tolerance 0.05 mm
+direction forward
+
+accept    13.4999969828397   54.9999995493206   -0.6034
+expect    13.4999906103972   54.9999957469562   -0.6374
+
+direction inverse
+
+accept    13.4999906103972   54.9999957469562   -0.6374
+expect    13.4999969828397   54.9999995493206   -0.6034
+-------------------------------------------------------------------------------
+This example is a random point, transformed from ED50 to ETRS89 using KMStrans2.
+-------------------------------------------------------------------------------
+operation proj=latlong ellps=intl
+          towgs84 =  -081.07030, -089.36030, -115.75260,
+                     -000.48488, -000.02436, -000.41321, -0.540645
+-------------------------------------------------------------------------------
+tolerance  0.2 mm
+accept     16.8210462130   55.1705688946       29.0317
+#          3494994.30120000 1056601.97250000 5212382.16660000
+expect     16.819999997    55.170000002        61.0
+#          3494909.84026368 1056506.78938633 5212265.66699761
+-------------------------------------------------------------------------------
+
+
+
+-------------------------------------------------------------------------------
+operation  proj=latlong nadgrids=nzgd2kgrid0005.gsb  ellps=GRS80
+-------------------------------------------------------------------------------
+tolerance 1 nm
+accept    173                 -45                  0 0
+expect    172.999892181021551 -45.001620431954613  0 0
+direction inverse
+accept    172.999892181021551 -45.001620431954613  0 0
+expect    173                 -45                  0 0
+-------------------------------------------------------------------------------
+
+
+
+-------------------------------------------------------------------------------
+operation  proj=latlong geoidgrids=egm96_15.gtx  ellps=GRS80
+-------------------------------------------------------------------------------
+tolerance 15 cm      # lax tolerance due to widespread bad egm96 file
+accept    12.5 55.5   0
+expect    12.5 55.5 -36.0213
```

Comment:
With `cs2cs` I get the following output:

```
λ echo 55.5 12.5  0|bin\cs2cs.exe -f %.2f +proj=latlong +to +proj=latlong +geoidgrids=egm96_15.gtx  +axis=neu +ellps=GRS80
12.50   55.50 42.37

C:\dev\build\ninjaproj
λ echo 55.5 12.5  0|bin\cs2cs.exe -f %.2f +proj=latlong +geoidgrids=egm96_15.gtx  +axis=neu +ellps=GRS80 +to +proj=latlong
12.50   55.50 36.02
```
gie returns:

```
     -----
     FAILURE in 4D-API_cs2cs-style.gie(94):
     expected: 12.5 55.5 -36.0213
     got:      12.500000000   55.500000000   42.372592926
     deviation:  78393.893 mm,  expected:  150.000 mm
``` 

which corresponds to the first call to `cs2cs` above. I guess this all comes down to what we mean by `+axis`. Does `+axis` describe the ordering of the input or the output coordinates?